app/src/login.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./login.css";
+
+async function loginUser(credentials) {
+  return fetch("http://localhost:1000/login", {
+    method: "POST",
+
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(credentials),
+  }).then((response) => response.json());
+}
+
+function Login() {
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
+  const [response_message, setresponse_message] = useState("");
```

Comment:
```suggestion
  const [responseMessage, setResponseMessage] = useState("");
```
use camelCase notation in JS